docs/validator-tickets-calculation.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Validator Tickets Calculation
+slug: /protocol/validator-tickets-calculation
+---
+
+:::info 
+This page is a guide to help you understand how Validator Tickets (VTs) are calculated.
+:::
+
+### VT Consumption Calculation Summary
+
+The VT (Validator Tickets) consumption calculation is a critical process for managing node operators within the Puffer Protocol. This process ensures that node operators maintain sufficient VT balances to support their active and exiting validators. Here's a breakdown of how VT consumption is calculated:
+
+1. **Node Operator Address Validation**: The process begins by validating the node operator's address to ensure it is in the correct format.
+
+2. **Fetching VT Balance**: The current VT balance of the node operator is retrieved from the Puffer Protocol. This balance represents the total VTs available for consumption by the node operator's validators.
+
+3. **Epoch and Validator Data Retrieval**: The latest epoch is fetched from the beacon chain, along with the list of active validators and those that have exited but not yet finalized. Active validators consume VTs daily, while exited validators may still consume VTs until they are fully finalized.
+
+4. **VT Consumption Calculation**:
+   - **Active Validators**: The total VT consumption for active validators is calculated based on the number of epochs they have been active.
+   - **Exited Validators**: Similarly, the VT consumption for exited but not finalized validators is calculated for the epochs they were active until their exit.
+
+5. **Present VT Balance Calculation**: The present VT balance is determined by subtracting the total VT consumption from the initial balance. This balance indicates the remaining VTs available to the node operator.
+
+:::tip
+Basically 1 VT is consumed daily per validator.
+:::
+
+6. **Minimum VT Requirement Check**: The system checks if the present VT balance meets the minimum required VTs per validator. If the balance is below this threshold, the node operator's validators may be ejected to prevent further VT consumption.
```

Comment:
```suggestion
6. **Minimum VT Requirement Check**: The system checks if the present VT balance meets the minimum required VTs per validator. If the balance is below this threshold, **ALL** of the the node operator's validators will be ejected to prevent further VT consumption.
```